src/api/.rubocop.yml

```diff
@@ -148,6 +140,21 @@ Style/TernaryParentheses:
 # Checks for predicates than can be replaced by receiver.empty? and !receiver.empty?
 Style/ZeroLengthPredicate:
   Enabled: true
+
+##################### Performance ###############################
```

Comment:
Performance in the Rubocop default config file is between Lint and Rails. Not sure why but maybe our config file should be ordered in the same way. :thinking: